app/__init__.py

```diff
@@ -1,7 +1,8 @@
 from flask import Flask
-
-
+from .routes.routes import planet_bp
+#if using planets_path file for planets_bp it would look like the following
+#from .routes.planets_path import planet_bp
```

Comment:
👍 For the content of the comment, but also, remember to leave a space between the `#` and the comment content
```py
# if using planets_path file for planets_bp it would look like the following
# from .routes.planets_path import planet_bp
```